torch/distributions/constraints.py

```diff
@@ -634,7 +634,7 @@ def check(self, value):
         sym_check = super().check(value)
         if not sym_check.all():
             return sym_check
-        return torch.linalg.cholesky_ex(value).info.eq(0)
+        return torch.linalg.cholesky_ex(value)[1].eq(0)
```

Comment:
Why the change here?

Reply:
I was seeing ` torch/distributions/constraints.py:637:16: error: "tuple[Tensor, Tensor]" has no attribute "info"  [attr-defined] `. Using this tuple access resolved the mypy errror, but the better solution is to keep `.info` and update `torch/_C/__init__.pyi.in`  to use `return_types_linalg_cholesk_ex` when applying the type checking.